aws_iam.tf

```diff
@@ -21,30 +24,73 @@ data "aws_iam_policy_document" "openvpn_eip" {
       "ec2:DescribeAddresses",
       "ec2:AssociateAddress",
       "ec2:AllocateAddress",
+      "kms:CreateGrant"
     ]
 
     resources = ["*"]
     effect    = "Allow"
   }
+
+  statement {
+    actions = [
+      "s3:ListBucket",
+      "s3:HeadBucket",
+      "s3:ListObjects",
+      "s3:GetObject"
+    ]
+
+    resources = ["arn:aws:s3:::*"]
```

Comment:
We should lock this down to only the buckets OVPN requires access to

Reply:
Roger that, I'll lock it down